src/classes/CaptureGroup.cls

```diff
@@ -0,0 +1,4 @@
+public interface CaptureGroup {
+    UTIL_Http.Response add(ElevateTokenizedGift tokenizedGift, String groupId);
+    UTIL_Http.Response create();
```

Comment:
`create` should return an `CaptureGroupId` (or at least a String, but not a full http response)